### PR TITLE
Display students by grade in descending order

### PR DESCRIPTION
**Pull request by JiaXinEu**

### CODE REVIEW

File: src/main/java/seedu/teachstack/model/person/UniquePersonList.java [hunk 1]
```diff
@@ -82,6 +86,7 @@ public void remove(Person toRemove) {
     public void setPersons(UniquePersonList replacement) {
         requireNonNull(replacement);
         internalList.setAll(replacement.internalList);
+        sort();
```
Comment (yashma-sonara) [inline src/main/java/seedu/teachstack/model/person/UniquePersonList.java:89]: Would it be better to have sort students in display work through a command? Or is the default behavior you wanted.

Comment (JiaXinEu) [reply src/main/java/seedu/teachstack/model/person/UniquePersonList.java]: I suppose this is the default behavior

**Review by yashma-sonara — APPROVED** [review]
LGTM

File: src/main/java/seedu/teachstack/logic/commands/DeleteCommand.java [hunk 2]
```diff
@@ -36,17 +33,12 @@ public DeleteCommand(StudentId targetId) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        List<Person> lastShownList = model.getAddressBook().getPersonList();
-
-        Optional<Person> personOptional = lastShownList.stream()
-                .filter(p -> p.getStudentId().equals(targetId))
-                .findFirst();
+        Person personToDelete = model.getPerson(targetId);
```
Comment (willowisp01) [inline src/main/java/seedu/teachstack/logic/commands/DeleteCommand.java:36]: This (and similar changes) look like a more concise way of obtaining the relevant person. Nicely done!

**Review by willowisp01 — APPROVED** [review]

**Pull request merged**